### ISSUE TITLE
Sample tests give plugin error

### ISSUE THREAD

**Issue by noelrappin** [cypress-io/cypress#15205]
### Current behavior
Running the default sample tests on Cypress 6.5 and Firefox 86 gives the following stack trace:

```
TypeError: SizeOnlySource is not a constructor
    at updateFileWithReplacementSource (/Users/noelrappin/Noel/north_by/node_modules/webpack/lib/Compiler.js:700:37)
    at updateWithReplacementSource (/Users/noelrappin/Noel/north_by/node_modules/webpack/lib/Compiler.js:682:8)
    at processExistingFile (/Users/noelrappin/Noel/north_by/node_modules/webpack/lib/Compiler.js:716:8)
    at /Users/noelrappin/Noel/north_by/node_modules/webpack/lib/Compiler.js:778:10
    at callback (/Users/noelrappin/Noel/north_by/node_modules/graceful-fs/polyfills.js:295:20)
    at callback (/Users/noelrappin/Library/Caches/Cypress/6.5.0/Cypress.app/Contents/Resources/app/packages/server/node_modules/graceful-fs/polyfills.js:289:20)
    at FSReqCallback.oncomplete (fs.js:173:5)
```

```
Error: The following error was thrown by a plugin. We stopped running your tests because a plugin crashed. Please check your plugins file (`/Users/noelrappin/Noel/north_by/cypress/plugins/index.js`)
    at Object.get (/Users/noelrappin/Library/Caches/Cypress/6.5.0/Cypress.app/Contents/Resources/app/packages/server/lib/errors.js:966:15)
    at EventEmitter.handleError (/Users/noelrappin/Library/Caches/Cypress/6.5.0/Cypress.app/Contents/Resources/app/packages/server/lib/plugins/index.js:168:20)
    at EventEmitter.emit (events.js:315:20)
    at ChildProcess.<anonymous> (/Users/noelrappin/Library/Caches/Cypress/6.5.0/Cypress.app/Contents/Resources/app/packages/server/lib/plugins/util.js:19:22)
    at ChildProcess.emit (events.js:315:20)
    at emit (internal/child_process.js:876:12)
    at processTicksAndRejections (internal/process/task_queues.js:85:21)
```

### Desired behavior
Sample tests run

### Test code to reproduce
Any sample test 

### Versions

Cypress 6.5, MacOS 11.2.1

**Comment by jennifer-shehane** [comment]
You are running the kitchen sink tests that are created when you start a project?

Do you have any preprocessors, babelrc, typescript settings? Anything in the plugins/index.js file?

If you can provide a repo, that would be helpful. Thanks.

**Comment by noelrappin** [comment]
Yes, kitchen sink tests. Nothing in the plugin/index.js file beyond the default. I haven't written any Cypress code, just running the kitchen sink tests.

it's a rails project running webpacker and cypress-rails, it's using TypeScript for other things, but not anything here yet. It's a fairly stock setup, and it does work in Cypress 4.12, which is the version I was using previously.

**Comment by jennifer-shehane** [comment]
We try to support TypeScript by default and sometimes that can cause some errors trying to serve the files. Can you provide a repo with a reproducible example? Thanks!

**Comment by noelrappin** [comment]
Is it possible this is related to using Webpack 5? I can try to give you a repo, but the actual repo is private.

**Comment by Big-Gray-Al** [comment]
Hi there,
Confirming the issue is still present today. The code is located here 
https://media.pragprog.com/titles/nrclient/code/nrclient-code.zip
with the project root located at code/chapter_15/01
If ruby is on your machine you can setup the app by calling bin/setup

**Comment by Big-Gray-Al** [comment]
It doesnt seem to work with the code from the previous chapter 14/04 and again raises even with "6.0.0.rc6"
`TypeError: SizeOnlySource`

**Comment by GrumpyPirate** [comment]
Chiming in on this, as our team had the same issue recently and hoping it helps others.

We had a base `tsconfig` (i.e. separate from the cypress tsconfig) that had the `paths` configuration customised. For whatever reason, this was causing the `SizeOnlySource is not a constructor` issue when opening cypress. We've reworked our `paths` config (frankly we just removed it, it wasn't being used anyway), and this resolved the issue.

We have webpack 5 explicitly installed as a dependency in our package.json. [`SizeOnlySource` appears to be a webpack internal](https://github.com/webpack/webpack/blob/747c20b9c0d860052bb7952408836d00a56e8776/declarations.d.ts#L340), so I can only assume our base tsconfig was messing with imports in some way.
